### PR TITLE
Override LEND logo

### PR DESCRIPTION
**Pull request by pablojorge**
Removed from upstream

### CODE REVIEW

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Review by mpfluger — APPROVED** [review]

File: erc20-tokens.json [hunk 1]
```diff
@@ -5674,7 +5674,7 @@
     {
         "address": "0x80fB784B7eD66730e8b1DBd9820aFD29931aab03",
         "decimals": 18,
-        "logo": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x80fB784B7eD66730e8b1DBd9820aFD29931aab03/logo.png",
```
Comment (Kalliingi) [inline erc20-tokens.json:5677]: @

File: erc20-tokens.json [hunk 2]
```diff
@@ -5674,7 +5674,7 @@
     {
         "address": "0x80fB784B7eD66730e8b1DBd9820aFD29931aab03",
         "decimals": 18,
```
Comment (Kalliingi) [inline erc20-tokens.json:5676]: @

**Review by Kalliingi — COMMENTED** [review]
Hope im not fucking up

**Pull request merged**